dora-asm/src/x64.rs

```diff
@@ -571,6 +571,26 @@ impl AssemblerX64 {
         self.emit_modrm_sse_registers(dest, src);
     }
 
+    pub fn roundss_ri(&mut self, reg: XmmRegister, imm: Immediate) {
+        self.emit_u8(0x66);
+        self.emit_rex_sse_modrm_optional(reg, reg);
+        self.emit_u8(0x0f);
+        self.emit_u8(0x3a);
+        self.emit_u8(0x0a);
+        self.emit_modrm(0b11, reg.low_bits(), reg.low_bits());
```

Comment:
You can use `emit_modrm_sse_registers` here instead.

Reply:
thanks, will do

Reply:
done!